Wikipedia/Code/WMFSearchViewController.m

```diff
@@ -501,6 +507,9 @@ - (void)updateLanguageButtonsToPreferredLanguages {
 - (void)resizeLanguageButtonsIfNeeded {
     [self.languageButtons enumerateObjectsUsingBlock:^(UIButton* _Nonnull obj, NSUInteger idx, BOOL* _Nonnull stop) {
         [obj sizeToFit];
+        CGRect f = obj.frame;
+        f.size.height = 30.0f;
+        obj.frame = f;
```

Comment:
This height constraint should be placed in the same place the height is currently set - that is probably in the storyboard. But it might be in the button subclass.